docs/content/preview/integrations/cdc/debezium.md

```diff
@@ -123,6 +123,44 @@ curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json"
 }'
 ```
 
+{{< note title="Note" >}}
+
+If you have an SSL enabled cluster, you need to provide the path to the root certificate in the `database.sslrootcert` configuration property.
+
+{{< /note >}}
+
+**In case of SSL enabled YugabyteDB cluster, follow these steps:**
+
+* Copy the certificate file to your Docker container (assuming that the file exists on the root directory of your machine)
+
+```sh
+docker cp ~/root.crt connect:/kafka/
+```
+
+* Deploy the connector configuration
+
+```sh
+curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json" \
+  localhost:8083/connectors/ \
+  -d '{
+  "name": "ybconnector",
+  "config": {
+    "connector.class": "io.debezium.connector.yugabytedb.YugabyteDBConnector",
+    "database.hostname":"'$IP'",
+    "database.port":"5433",
+    "database.master.addresses": "'$IP':7100",
+    "database.user": "yugabyte",
+    "database.password": "yugabyte",
+    "database.dbname" : "yugabyte",
+    "database.server.name": "dbserver1",
+    "table.include.list":"public.test",
+    "database.streamid":"d540f5e4890c4d3b812933cbfd703ed3",
+    "snapshot.mode":"never",
+    "database.sslrootcert":"/kafka/root.crt"
+  }
+}'
+```
```

Comment:
Bulleted lists imply that ordering is not significant. This is a procedure, so we need to number the steps.
```suggestion
If you have an SSL enabled YugabyteDB cluster, do the following:

1. Copy the certificate file to your Docker container (assuming that the file exists on the root directory of your machine):

    ```sh
    docker cp ~/root.crt connect:/kafka/
    ```

1. Deploy the connector configuration:

    ```sh
    curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json" \
      localhost:8083/connectors/ \
      -d '{
      "name": "ybconnector",
      "config": {
        "connector.class": "io.debezium.connector.yugabytedb.YugabyteDBConnector",
        "database.hostname":"'$IP'",
        "database.port":"5433",
        "database.master.addresses": "'$IP':7100",
        "database.user": "yugabyte",
        "database.password": "yugabyte",
        "database.dbname" : "yugabyte",
        "database.server.name": "dbserver1",
        "table.include.list":"public.test",
        "database.streamid":"d540f5e4890c4d3b812933cbfd703ed3",
        "snapshot.mode":"never",
        "database.sslrootcert":"/kafka/root.crt"
      }
    }'
    ```
```